tests/providers/test_misc.py

```diff
@@ -477,18 +477,22 @@ def test_json_multiple_rows(self, faker_with_foobar):
 
         assert isinstance(json_data, list) and len(json_data) == 2
 
-    def test_json_passthrough_int_float(self, faker_with_foobar):
+    def test_json_passthrough_values(self, faker_with_foobar):
         kwargs = {
             'data_columns': {
                 'item1': 1,
                 'item2': 1.0,
+                'item3': True,
+                'item4': '@fixed',
             },
             'num_rows': 1,
         }
         json_data = json.loads(faker_with_foobar.json(**kwargs))
 
         assert json_data['item1'] == 1
         assert json_data['item2'] == 1.0
+        assert json_data['item3']
```

Comment:
I would change add `is True` just for clarity purposes here.

```suggestion
        assert json_data['item3'] is True
```

Reply:
Agreed, that was a bit of a miss by me.  It's there now.